rezz.lic

```diff
@@ -69,10 +69,10 @@ class Rezz
   end
 
   def rejuv(player, quick = false)
-    @rejuv = { 'abbrev' => 'rejuv', 'prep_time' => 5, 'cast' => "cast #{player}", 'mana' => 10 } if quick
+    quick ? spell = { 'abbrev' => 'rejuv', 'prep_time' => 5, 'cast' => "cast #{player}", 'mana' => 10 } : spell = @rejuv
```

Comment:
💡 This could be `spell =  quick ? { 'abbrev' => 'rejuv', 'prep_time' => 5, 'cast' => "cast #{player}", 'mana' => 10 } : @rejuv`

Reply:
Thanks for the coding tip. :-)